requirements.txt

```diff
@@ -18,18 +18,17 @@ fprime-fpl-convert-xml==1.0.3
 fprime-fpl-extract-xml==1.0.3
 fprime-fpl-layout==1.0.3
 fprime-fpl-write-pic==1.0.3
-fprime-fpp-check==2.1.0a7
-fprime-fpp-depend==2.1.0a7
-fprime-fpp-filenames==2.1.0a7
-fprime-fpp-format==2.1.0a7
-fprime-fpp-from-xml==2.1.0a7
-fprime-fpp-locate-defs==2.1.0a7
-fprime-fpp-locate-uses==2.1.0a7
-fprime-fpp-syntax==2.1.0a7
-fprime-fpp-to-cpp==2.1.0a7
-fprime-fpp-to-dict==2.1.0a7
-fprime-fpp-to-json==2.1.0a7
-fprime-fpp-to-xml==2.1.0a7
+fprime-fpp-check==2.1.0a9
+fprime-fpp-depend==2.1.0a9
+fprime-fpp-filenames==2.1.0a9
+fprime-fpp-format==2.1.0a9
+fprime-fpp-from-xml==2.1.0a9
+fprime-fpp-locate-defs==2.1.0a9
+fprime-fpp-locate-uses==2.1.0a9
+fprime-fpp-syntax==2.1.0a9
+fprime-fpp-to-cpp==2.1.0a9
+fprime-fpp-to-json==2.1.0a9
+fprime-fpp-to-xml==2.1.0a9
```

Comment:
`fprime-fpp-to-dict` should be added back, it probably got removed because this branch is older than the one that introduced it

Reply:
Good catch! Fixed.